stagemonitor-web-servlet/src/main/java/org/stagemonitor/web/servlet/MonitoredHttpRequest.java

```diff
@@ -254,6 +270,10 @@ public void onFinish(SpanWrapper span, String operationName, long durationNanos)
 					logger.warn("Suppressed exception", e);
 				}
 			}
+			final Long gcTime = contextInfo.getRequestAttribute(GC_TIME_ATTRIBUTE);
```

Comment:
Move this to a SpanEventListener. That way it works for other request types as well.